tests/dragonfly/conftest.py

```diff
@@ -28,6 +28,30 @@
 logging.getLogger("asyncio").setLevel(logging.WARNING)
 
 DATABASE_INDEX = 0
+BASE_LOG_DIR = "/tmp/dragonfly_logs/"
+
+
+def pytest_configure(config):
+    if os.path.exists(BASE_LOG_DIR):
+        shutil.rmtree(BASE_LOG_DIR)
+
+
+def pytest_runtest_setup(item):
```

Comment:
Same here, plz add a comment of what pytest_runtest_setup is doing (I now know that it runs per test case but other people in the team might not know this)